### PR TITLE
update term to spring 2025

### PR DESCRIPTION
**Pull request by dtemkin1**
updates term to spring 2025. archives fall 2024. removes fall overrides (can be added later).

### CODE REVIEW

**Comment by dtemkin1** [comment]
@psvenk

**Comment by cjquines** [comment]
https://github.com/sipb/hydrant/issues/7 rears its head again

**Comment by psvenk** [comment]
Thanks for opening this; I think we should make sure the fall 2024 data on the FireRoad server are fully up to date (as of the archived copy of the fall 2024 subject listing) and committing the results of that as `f24.json` before switching Hydrant to spring 2025.

File: public/latestTerm.json [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
-  "startDate": "2024-09-04",
-  "h1EndDate": "2024-10-18",
-  "h2StartDate": "2024-10-21",
-  "endDate": "2024-12-11",
+  "urlName": "s25",
+  "startDate": "2025-02-03",
+  "h1EndDate": "2025-03-21",
+  "h2StartDate": "2025-03-31",
+  "endDate": "2025-05-13",
+  "mondayScheduleDate": "2025-02-18",
```
Comment (gabrc52) [inline public/latestTerm.json:7]: Dates look good

**Review by gabrc52 — APPROVED** [review]

**Comment by gabrc52** [comment]
@psvenk can you merge once you update Fireroad?

**Review by psvenk — APPROVED** [review]

**Pull request merged**